src/index.html

```diff
@@ -15,8 +15,99 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="site_header">
+      <a
+        class="logo"
+        href="index.html"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav_li">
```

Comment:
fix class everywhere
```suggestion
          <li class="nav__item">
```